panel/viewable.py

```diff
@@ -251,6 +251,23 @@ def __init__(self, **params):
             elif config.sizing_mode == 'stretch_height' and 'height' not in params:
                 params['sizing_mode'] = config.sizing_mode
         super().__init__(**params)
+        self._set_background()
+        self.param.watch(self._set_background, 'background')
```

Comment:
@philippjfr this line makes `test_param_function_pane_update` fails. Even if I replace it with something like `self.param.watch(lambda x: print(x), 'background')`. I can recreate it on the current master.  Do you know why it this is the case?

``` python
document = <bokeh.document.document.Document object at 0x7f6569a7a4d0>, comm = Comm(id='c5831af2ef354ca39bcc1b228eabc6d5', name='Comm01251')

    def test_param_function_pane_update(document, comm):
        test = View()
    
        objs = {
            0: HTML("012"),
            1: HTML("123")
        }
    
        @param.depends(test.param.a)
        def view(a):
            return objs[a]
    
        pane = panel(view)
        inner_pane = pane._pane
>       assert inner_pane is not objs[0]
E       assert HTML(str) is not HTML(str)

panel/tests/test_param.py:1109: AssertionError

```

Reply:
Same thing with these tests:
```
FAILED panel/tests/io/test_embed.py::test_embed_param_jslink - KeyError: 'target'
FAILED panel/tests/io/test_embed.py::test_embed_select_str_jslink - AssertionError: assert 2 == 1
FAILED panel/tests/io/test_embed.py::test_embed_checkbox_str_jslink - AssertionError: assert 2 == 1
FAILED panel/tests/io/test_embed.py::test_embed_slider_str_jslink - AssertionError: assert 2 == 1
```

Reply:
Found a workaround in dec2eb7a29334e11416f70d8db09f6818d616f23